server/settings/conversion.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any
+
+
+def _convert_collect_shots_plugins_1_1_0(
+    overrides: dict[str, Any]
+):
+    """Report settings from "CollectTimelineInstances"
+    to "CollectShot".
+
+    1.0.0 is the latest version using the old way
+    """
+    if "CollectTimelineInstances" not in overrides["publish"]:
```

Comment:
This would crash if there are no overrides are for `"publish"` key.
```suggestion
    publish_overrides = overrides.get("publish", {})
    if "CollectTimelineInstances" not in publish_overrides:
```

Reply:
Thanks for the recommendation.